modules/broker-core/src/main/java/org/wso2/broker/core/Broker.java

```diff
@@ -111,4 +121,26 @@ public long getNextMessageId() {
     public void requeue(String queueName, Message message) {
         messagingEngine.requeue(queueName, message);
     }
+
+    @Override
+    protected void onBecameActiveNode() {
+        if (!isActive.getAndSet(true)) {
+            //if state changed from passive to active
+            startMessageDelivery();
+            log.info("Broker is now in ACTIVE mode.");
```

Comment:
Maybe something on the lines of `Broker became the active node`? Since the two log messages are too similar